src/olympia/scanners/models.py

```diff
@@ -48,4 +51,24 @@ def save(self, *args, **kwargs):
 
     @property
     def matched_rules(self):
+        """This method returns a sorted list of matched rule names."""
         return sorted({match['rule'] for match in self.matches})
+
+
+class ScannerRule(ModelBase):
+    name = models.CharField(
+        max_length=200,
+        unique=True,
+        help_text=_('This is the exact name of the rule used by a scanner.'),
+    )
+    scanner = models.PositiveSmallIntegerField(choices=SCANNERS.items())
+    action = models.PositiveSmallIntegerField(
+        choices=ACTIONS.items(), default=NO_ACTION
+    )
+    is_active = models.BooleanField(default=True)
+
+    class Meta:
+        db_table = 'scanners_rules'
```

Comment:
Note: it's perfectly fine to do it, but you don't need to bother with setting `db_table` explicitly every time you create a model - the default is fine. We've got a lot of those hanging around because many of our tables are older than the django app :)